packages/components/src/index.ts

```diff
@@ -1,35 +1,37 @@
-import { FrameworkConfiguration, PLATFORM } from 'aurelia-framework';
```

Comment:
Shouldn't this file not affected by the line ending change any more? Can you help merge master into this branch

Reply:
eols fixed. I've merged the master yesterday after another PR got accepted